data/authors/philniedertscheider.md

```diff
@@ -0,0 +1,10 @@
+---
+name: Phil Niedertscheider
+avatar: /static/avatars/philniedertscheider.jpg
+occupation: Senior Software Engineer
+url: https://philprime.dev
+github: https://github.com/philprime
+---
+
+Phil's focus at Sentry is on Apple platforms, maintaining the Cocoa SDK, building mobile apps and crafting tools to make developers' lives easier.
```

Comment:
We were dropping the use of Cocoa in write ups because that's not used as much these days
```suggestion
Phil's focus at Sentry is on Apple platforms, maintaining the Apple SDK, building mobile apps and crafting tools to make developers' lives easier.
```